lib/visitor.js

```diff
@@ -333,6 +288,22 @@ class Visitor {
         this.contexts.pop()
     }
 
+    /**
+     * @param {EntityCSN} entity -
+     * @returns {string[]}
+     */
+    #getInheritedElements(entity) {
```

Comment:
This smells a bit like a bottle neck (both, the finding of these elements and using an array over, say, `Set`) and something that could be cached as part of `EntityInfo`. But we can probably benchmark it at some later point.

Reply:
Moved into entity repository